### ISSUE TITLE
AttributeError: 'BERTPreprocessor' object has no attribute 'ytransform' 

### ISSUE THREAD

**Issue by FatimaMoqran** [amaiya/ktrain#227]
hello!
I trained successfully a Bert model and I saved the model:
```
#model saved
predictor.save('/content/drive/My` Drive/notebooks_chefdoeuvre_deep/model_corps_mail_bert)
```
The last week it was working but today when I tried to predict the data : 

```
#loading the model
predictor= ktrain.load_predictor('/content/drive/My Drive/notebooks_chefdoeuvre_deep/model_corps_mail_bert')
data2 = 'something'
predictor.predict(data2)
```
I get this error 
`AttributeError: 'BERTPreprocessor' object has no attribute 'ytransform' `

I am on google colab .

Thank you for your help!!
```

/usr/local/lib/python3.6/dist-packages/ktrain/text/preprocessor.py in _transform_y(self, y_data, train, verbose)
    502         or regression if not.
    503         """
--> 504         if self.ytransform is None:
    505             self.ytransform = U.YTransform(class_names=self.get_classes())
    506         return self.ytransform.apply(y_data, train=train)

AttributeError: 'BERTPreprocessor' object has no attribute 'ytransform'
```

**Comment by amaiya** [comment]
Thanks a lot for reporting this issue.  

A change was made in v0.19.4 that caused a backwards compatibility issue.  This is now fixed in the latest version of *ktrain* that was just released (v0.19.5).  So, if you re-run those steps with the latest version of *ktrain*, everything should now work.

**Comment by FatimaMoqran** [comment]
ok thank you!
